### PR TITLE
Games [NEW]

### PR DESCRIPTION
**Pull request by shuvashish76**
Tachiyomi's F-Droid link :)

Edit: @offa  keep it open. I'll keep adding Games in this pr & let you know when done.

### CODE REVIEW

**Comment by offa** [comment]
> Edit: @offa keep it open. I'll keep adding Games in this pr & let you know when done.

All right! I've changed the PR to draft, so you can mark it as ready _for review_ at any time when ready. :-)

**Comment by offa** [comment]
> Tachiyomi's F-Droid link :)

This topic seems a little complicated https://github.com/offa/android-foss/pull/195

**Comment by shuvashish76** [comment]
> 

Ok, removed :)

**Comment by shuvashish76** [comment]
Please do add a new "**Games**" Label.

File: README.md [hunk 1]
```diff
@@ -420,6 +433,70 @@ Table of Contents
...
+- [**TriPeaks**](https://github.com/mimoguz/tripeaks-gdx) <sup>**[[F-Droid](https://f-droid.org/app/ogz.tripeaks)]**</sup>
+- [**MTG Familiar**](https://github.com/AEFeinstein/mtg-familiar) <sup>**[[F-Droid](https://f-droid.org/app/com.gelakinetic.mtgfam)]**</sup>
+
+
+### • Educational
+- [**GCompris**](https://invent.kde.org/education/gcompris.git) <sup>**[[F-Droid](https://f-droid.org/app/net.gcompris.full)]**</sup>
+
+
+### • Miscellaneous
+- [**Zoysii**](hhttps://gitlab.com/deepdaikon/Zoysii) <sup>**[[F-Droid](https://f-droid.org/app/xyz.deepdaikon.zoysii)]**</sup>
```
Comment (offa) [inline README.md:462]: There's an extra `h` which breaks the link. I'll fix that up when updating the ToC.

File: README.md [hunk 2]
```diff
@@ -420,6 +433,70 @@ Table of Contents
 - [**Forecastie**](https://github.com/martykan/forecastie) <sup>**[[F-Droid](https://f-droid.org/app/cz.martykan.forecastie)]**</sup>
 
 
+------------------------
+
+
+## – Games –
+
+### • Board
+- [**lichess**](https://github.com/lichess-org/lichobile)
```
Comment (offa) [inline README.md:442]: Includes firebase unfortunately (related: https://github.com/lichess-org/lichobile/issues/1121).

**Review by offa — COMMENTED** [review]
Thanks! :+1: 

Looks good. There are two broken links and a proprietary app. Since the former is easy to fix I'm going to merge the PR and drop *lichess* so it wont block the PR.

Feel free to open a PR or issue related to lichess.

File: README.md (hunk 2 above)
Comment (offa) [inline README.md:442]:
```suggestion
```

**Pull request merged**